omnibus/files/private-chef-cookbooks/private-chef/attributes/default.rb

```diff
@@ -490,6 +490,7 @@
 default['private_chef']['nginx']['ssl_organizational_unit_name'] = 'Operations'
 default['private_chef']['nginx']['ssl_key_length'] = 2048
 default['private_chef']['nginx']['ssl_duration'] = 3650
+default['private_chef']['nginx']['sts_max_age'] = 31536000
```

Comment:
Should there be a warning issues if the user sets this value to be less than a year or more than 2 years perhaps?

Reply:
Made the changes to limit the max-age to be within 1 and 2 years. If the value is out of the range or invalid data is given the default value is 1 year.

Reply:
nice!